usecase/usecase-webapi-xwalk-tests/tests.android.xml

```diff
@@ -3,143 +3,143 @@
 <test_definition>
   <suite name="Usecase WebAPI">
```

Comment:
Yes, we also need update the suite name, but this will affect display name of title in usecase, this is why we change it. @jennycao What's your opinion?